Jenkinsfile

```diff
@@ -1,5 +1,12 @@
 @Library('xmos_jenkins_shared_library@v0.24.0') _
 
+def buildApps(appList) {
```

Comment:
This function is helpful. @danielpieczko I wonder if a similar script couldn't be added to the tools to make use of cmake xcommon even easier?

Reply:
Yes this is the kind of thing I would expect we could add to the jenkins shared library.